xpanse/utils.py

```diff
@@ -11,3 +13,47 @@ def normalize_param_names(kwargs: Dict[str, Any]):
             normalized_args[k.replace("_neq", "!").replace("_", "-")] = v
         kwargs["params"] = normalized_args
     return kwargs
+
+
+def build_request_payload(
+    request_data: Optional[dict] = None,
```

Comment:
I think we could simplify this and [this line](https://github.com/PaloAltoNetworks/cortex-xpanse-python-sdk/pull/127/files#diff-024fe40414f72388e33af1ee901ffda0b8be7eee8dde312aa6a33803e07855d0R42) if we made default = {}

Reply:
I think that this would make the field mutable since it's only instantiated once. I think we would get side effects on the `request_data` object

Reply:
good point - nice doc, too, going to bookmark that one!